Automattic/wp-calypso
# Site migration: add View Site button to homepage banner

#### Changes proposed in this Pull Request

* Adds a View Site button to the banner we show at the top of the Customer Home on a site which has been migrated to in the last two days. ([Related PR](https://github.com/Automattic/wp-calypso/pull/39564))

#### Testing instructions

* Apply the PR and run Calypso locally, or use calypso.live (for the latter, add `?flags=tools/migrate` to the end of the URL to enable the feature flag).
* Do a migration: go to the import section of a Simple site, enter the URL of a Jetpack site and follow the steps you're presented with. (If you're migrating to a site you've migrated to whose `done` status hasn't been cleared yet, you may need to reset the migration first with a request to the `reset-migration` endpoint.)
* When the migration finishes you should be redirected to My Home, unless there was an error.
* The "Your site has been imported!" banner at the top of My Home should include a "View Site" button.

<img width="1279" alt="image" src="https://user-images.githubusercontent.com/1647564/75172012-7add8b00-5724-11ea-94e1-c814d574dbac.png">

<img width="377" alt="image" src="https://user-images.githubusercontent.com/1647564/75172032-84ff8980-5724-11ea-8cbf-567c57f94f11.png">
